documental/admin.py

```diff
@@ -34,23 +35,23 @@ class DocumentalDocsAdmin(admin.ModelAdmin):
     """Django administrator `model.DocumentalDocs` data."""
 
     list_display = (
-        'id',
-        'path_documento',
-        'no_documento',
-        'usuario_id',
-        'st_disponivel',
-        'st_excluido',
-        'dt_cadastro',
-        'dt_atualizacao',
+        'id_document',
+        'path_document',
+        'no_document', 
```

Comment:
Trailing whitespace.

Please use pep8 to format this file.

Reply:
Removed.